.pre-commit-config.yaml

```diff
@@ -11,6 +11,7 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
+        args: [--max-line-length=88, --extend-ignore=E203]
```

Comment:
AFAIK that has to be set in the `setup.cfg` or in the `pyptoject.toml`.

I am a bit surprised you need this.
It works for me like it is now :)
```
ale@code:~/Code/plone/projects/eu5rie/src/Euphorie$ pre-commit run flake8 --all
flake8...................................................................Passed
```

Reply:
Ah, now it makes sense. This first came up when I reused the pre-commit config for oira.application.buildout. This change is actually not needed if I update one of the files you mentioned in the other repo.